test/utils/schema.sql

```diff
@@ -14,7 +14,9 @@ CREATE TABLE users (
   surname TEXT NOT NULL,
   password TEXT NOT NULL,
   email TEXT NOT NULL UNIQUE,
-  photo_url TEXT NOT NULL DEFAULT 'https://s3.eu-west-2.amazonaws.com/spark-native/avatar.png'
+  photo_url TEXT NOT NULL DEFAULT 'https://s3.eu-west-2.amazonaws.com/spark-native/avatar.png',
+  reset_password_token TEXT,
```

Comment:
I wonder if this might be better as JSON, as an enhancement?  So maybe one column called `reset_password` with `{ token: null, expires: null }`?  Don't have to worry about it now though.